packages/l/libgudev/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 7735
+  rss: https://gitlab.gnome.org/GNOME/libgudev/-/releases.atom
```

Comment:
Gitlab doesn't use the same format as github. For this particular package, it doesn't have a release page but do have tags page. This is the correct one:
```
rss: https://gitlab.gnome.org/GNOME/libgudev/-/tags?format=atom
```